example-app/src/app/screens/management/OwnerCards.tsx

```diff
@@ -49,7 +49,8 @@ export function OwnerCards() {
 
   // Load the items from server
   const { loading, error, data } = useQuery(OWNER_LIST);
-  const items = deserialize(data?.ownerList);
+
+  const items = useMemo(() => deserialize(data?.ownerList), [data]);
```

Comment:
I think it's more correct to destructure `data`. This is also what ESLint suggests when you put empty deps array.

```suggestion
  const items = useMemo(() => deserialize(data?.ownerList), [data?.ownerList]);
```